reveal/run.js

```diff
@@ -88,8 +91,23 @@
     });
   }
 
-  function hasRemoteQueryParameter() {
-    return window.location.search.match(/[?&]remote[&]?/i);
+  function selectWidth(queryParams) {
+    if (queryParameterIsPresent(window.location.search, 'wide')) return 1413; // 16:9
+    else if (queryParameterIsPresent(window.location.search, 'wide10')) return 1272; // 16:10
+    else return 1124; // ~4:3
+  }
+
+  function cycleWideModes() {
+    var queryString = window.location.search;
```

Comment:
I think this variable does not pay its ticket.
The only thing it does is avoid using `window.location.search` as a global... But it really is a global, so why bother ?
Moreover it introduces an extra parameter on *all* the following method calls.
I would prefer if it were eliminated. The utility methods would then use the global directly.